examples/run-example-008.py

```diff
@@ -7,11 +7,14 @@
 NORM_EXPECT = (
     'Require Coq.Arith.Arith.\nRequire Top.A.\nRequire Top.C.\nRequire Top.B.\nRequire Top.D.\n\nImport Top.D.\n\nFail Check A.mA.axA.\n',
     'Require Stdlib.Arith.Arith.\nRequire Top.A.\nRequire Top.C.\nRequire Top.B.\nRequire Top.D.\n\nImport Top.D.\n\nFail Check A.mA.axA.\n',
+    'Require Stdlib.Arith.All.Stdlib.Arith.Arith.\nRequire Top.A.\nRequire Top.C.\nRequire Top.B.\nRequire Top.D.\n\nImport Top.D.\n\nFail Check A.mA.axA.\n',
 )
 
 GET_EXPECT = {
     'Coq.Arith.Arith': None,
+    'Coq.Arith.All.Coq.Arith.Arith': None,
     'Top.A': 'Module Export Top_DOT_A.\nModule Export Top.\nModule A.\nImport Coq.Arith.Arith.\nModule mA.\n  Section secA.\n    Axiom axA : Set.\n  End secA.\nEnd mA.\n\nModule mA2.\nEnd mA2.\n\nEnd A.\n\nEnd Top.\n\nEnd Top_DOT_A.\n',
+    'Top.A': 'Module Export Top_DOT_A.\nModule Export Top.\nModule A.\nImport Coq.Arith.All.Coq.Arith.Arith.\nModule mA.\n  Section secA.\n    Axiom axA : Set.\n  End secA.\nEnd mA.\n\nModule mA2.\nEnd mA2.\n\nEnd A.\n\nEnd Top.\n\nEnd Top_DOT_A.\n',
```

Comment:
This is not backwards compatible.  Instead do
```suggestion
    'Top.A': [
        'Module Export Top_DOT_A.\nModule Export Top.\nModule A.\nImport Coq.Arith.Arith.\nModule mA.\n  Section secA.\n    Axiom axA : Set.\n  End secA.\nEnd mA.\n\nModule mA2.\nEnd mA2.\n\nEnd A.\n\nEnd Top.\n\nEnd Top_DOT_A.\n',
        'Module Export Top_DOT_A.\nModule Export Top.\nModule A.\nImport Coq.Arith.All.Coq.Arith.Arith.\nModule mA.\n  Section secA.\n    Axiom axA : Set.\n  End secA.\nEnd mA.\n\nModule mA2.\nEnd mA2.\n\nEnd A.\n\nEnd Top.\n\nEnd Top_DOT_A.\n'
        ],
```
and either make all of the others lists as well, or adapt the code below to iterate over possibilities when the expectation is not `None` and not a `str`.

Reply:
Thanks, fixed.